.github/workflows/release.yml

```diff
@@ -86,4 +86,4 @@ jobs:
         with:
           bodyFile: 'CHANGELOG.md'
           name: ${{ env.VERSION }}
-          prerelease: ${{ contains(github.ref_name, '-') }}
+          prerelease: ${{ contains(github.ref_name, '-') }}
```

Comment:
don't drop final newlines

Reply:
Didn't drop and didn't add either. See original https://github.com/zone117x/MimeMapping/blob/master/.github/workflows/release.yml
You can fix newlines in another PR.